### ISSUE TITLE
Suggestion: do we need a better way of identifying "childhood diseases"?

### ISSUE THREAD

**Issue by celiot-IDM** [InstituteforDiseaseModeling/PACE-HRH#213]
There's a feature whereby a scenario can be identified as having steadily decreasing levels of childhood disease:

![image](https://user-images.githubusercontent.com/53099836/199774222-683d0e8b-7e43-4524-ac0d-639e4f893b99.png)

Currently tasks affected by decreasing childhood disease are defined to be those where RelevantPop == "1-4". Tasks with this population label are done less over time as the affected portion of the group population decreases.

This definition is problematic.
- It creates a hard-coded dependency on a specific label that users are allowed to change.
- What about sub-ranges, specifically "1 yo" and "2 yo"? Today these ranges don't pose a problem because for both of them StartingRateInPop == 1.0, which is our indicator that prevalence within the population group is not a factor (just the group population). 
 
I don't think we document either of these behaviors: (1) only tasks with RelevantPop == "1-4" are affected by the o_ChildDis_decr flag; and (2) StartingRateInPop == 1.0 turns off prevalence calculations for that task.

Potential solutions:
1. Document existing behavior and leave it at that.
2. Create a globalconfig.json setting that identifies "1-4" as the tag for tasks affected by decreasing childhood disease.
3. Add a new Tasks column which is TRUE if a task is affected by decreasing childhood disease.

**Comment by BHagedorn-IDM** [comment]
Good point. I will write the documentation for this as it is the current behavior. We could add an indicator column to TaskValues for both the o_MHIVTB and o_ChildDis but that makes them actually generic on/off switches and means the names are no longer necessarily applicable, etc. So I'm hesitant to make that change. I'd suggest that we leave as is with documentation for now, and see what our users say.

**Comment by celiot-IDM** [comment]
Works for me!

**Comment by BHagedorn-IDM** [comment]
Documentation edits made and awaiting merge approval.
